File: TravellerCreatorModels/RTTWorldgen/RTTWorldgenSubsector.cs
using TravellerCreatorModels.Interfaces;
using TravellerCreatorModels.Other;

namespace TravellerCreatorModels.RTTWorldgen;

public class RTTWorldgenSubsector : ISubsector
{
    public Coordinates? Coordinates { get; set; }
    public List<IHex> Hexes { get; set; } = new();
}

File: TravellerCreator/ClientApp/src/app/sector-creator/models/rtt-worldgen/rtt-worldgen.subsector.ts
import {ISubsector} from "../interfaces/subsector";
import {Coordinates} from "../other/coordinates";
import {RttWorldgenHex} from "./rtt-worldgen.hex";

export class RttWorldgenSubsector implements ISubsector{
  coordinates: Coordinates;
  hexes: RttWorldgenHex[];
}
